cmd/client/style.go

```diff
@@ -5,10 +5,10 @@ import (
 )
 
 var (
-	txtSuccess   = color.FgGreen.Render
-	txtDanger    = color.FgRed.Render
-	txtWarning   = color.FgYellow.Render
-	txtHighlight = color.New(color.FgMagenta, color.OpBold).Render
-	txtWhite     = color.FgWhite.Render
-	txtWhiteBold = color.New(color.FgWhite, color.OpBold).Render
+	TxtSuccess   = color.FgGreen.Render
```

Comment:
This isn't necessary, but I thought I would show you. 
`TxtSuccess` <= this is a public constant/var. Unless you explicitly want something to be private, we usually just make things public. 

`txtSuccess` <= this is private.